nuxeo-services/nuxeo-platform-usermanager-core/src/main/java/org/nuxeo/ecm/platform/usermanager/UserManagerImpl.java

```diff
@@ -114,7 +114,20 @@
 
     public static final String INVALIDATE_ALL_PRINCIPALS_EVENT_ID = "invalidateAllPrincipals";
 
-    private static final String USER_GROUP_CATEGORY = "userGroup";
+    /**
+     * @since 9.2
+     */
+    public static final String USER_GROUP_CATEGORY = "userGroup";
+
+    /**
+     * @since 9.2
+     */
+    public static final String ID_PROPERTY_KEY = "id";
+
+    /**
+     * @since 9.2
+     */
+    public static final String ANCESTOR_GROUPS_PROPERTY_KEY = "ancestorGroups";
```

Comment:
Please explain in the Javadoc that these constants are keys in the event context.